boto/s3/bucket.py

```diff
@@ -2451,3 +2452,70 @@ def get_bucket_objectlock_config(self, headers=None):
             response = response.encode('utf-8')
         xml.sax.parseString(response, h)
         return olc
+
+    def configure_inventory(self, id, inventory, headers=None):
```

Comment:
Do other methods in this file not use comments to describe what the method does and the arguments they take?

Reply:
Ok. Added explainations to each method and class to understand what method/class is used for and how those work. Please review again. Thanks.